roles/backup/tasks/postgres.yml

```
@@ -82,6 +82,15 @@
     resolvable_db_host: '{{ (awx_postgres_type == "managed") | ternary(awx_postgres_host + "." + ansible_operator_meta.namespace + ".svc.cluster.local", awx_postgres_host) }}'  # yamllint disable-line rule:line-length
   no_log: "{{ no_log }}"
 
+- name: Precreate database partitions
+  k8s_exec:
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    pod: "{{ awx_task_pod_name }}"
+    container: "{{ ansible_operator_meta.name }}-task"
```

`ansible_operator_meta.name` will resolve to the awxbackup resource not the awx resource's name

---

Right, you'll need to swap this out for:

```suggestion
    container: "{{ deployment_name }}-task"
```